solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/inbound/InboundXMLMessageListener.java

```diff
@@ -91,18 +92,18 @@ public void run() {
 				try {
 					receive();
 				} catch (RuntimeException | UnboundFlowReceiverContainerException e) {
-					logger.warn(String.format("Exception received while consuming messages from destination %s",
-							consumerDestination.getName()), e);
+					LOGGER.warn("Exception received while consuming messages from destination {}",
```

Comment:
this not going to log as expected:  e not logged


    /**
     * Log a message at the WARN level according to the specified format
     * and arguments.
     * 
     * <p>This form avoids superfluous object creation when the logger
     * is disabled for the WARN level. 
     *
     * @param format the format string
     * @param arg1   the first argument
     * @param arg2   the second argument
     */
    public void warn(String format, Object arg1, Object arg2);

Reply:
You can use parameterized logging with a throwable as the last parameter. See https://www.slf4j.org/faq.html#paramException .

For example, when I do:
```java
LOGGER.warn("foo {}", "bar", new RuntimeException("test error", new RuntimeException("cause")));
```

I get the output:
```java
2024-08-28T14:34:34.396-04:00  WARN   --- [           main] c.s.s.c.s.b.util.XMLMessageMapperTest    : foo bar

java.lang.RuntimeException: test error
Caused by: java.lang.RuntimeException: cause
```